active/0000-firefox-in-ci.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2016-12-04
+- RFC PR: 
+- Ember CLI Issue: 
+
+# Summary
+
+Replace PhantomJS with Firefox as the default browser for continuous integration testing.
```

Comment:
The major issue with that linked method is `sudo: required`. That prevents you from using Travis's faster, newer container-based infrastructure.